.github/workflows/release-angular.yml

```diff
@@ -46,23 +46,24 @@ jobs:
       - name: Set Tegel user
         run: git config --global user.name "Tegel - Scania" && git config --global user.email "tegel.design.system@gmail.com"
 
-      - name: Angular - CD into correct folder
-        run: cd packages/angular
-
       - name: Angular - Bump version
         run: npm version ${{ github.event.inputs.tegelVersion }}
+        working-directory: packages/angular
 
       - name: Angular - Remove package-lock
         run: rm -rf package-lock.json
```

Comment:
What was the reason for removing this? (just curious)

Reply:
Do you mean the "cd:ing" into the correct folder? That is all handled by the last step, the creation of the PR.

Reply:
Ah no what was the reason for removing package-lock.json?

Reply:
Ah, that was just due to us being **sure** we are using the latest @scania/tegel version as a base